xgboost-demo-all.ipynb

```diff
@@ -0,0 +1,222 @@
+    "\n",
+    "Thanks to this you can run this code as is and see results in Neptune :)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "neptune.init('shared/XGBoost-integration',\n",
+    "             api_token='ANONYMOUS')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare data for XGBoost training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "boston = load_boston()\n",
+    "data = pd.DataFrame(boston.data)\n",
+    "data.columns = boston.feature_names\n",
+    "data['PRICE'] = boston.target\n",
+    "X, y = data.iloc[:,:-1], data.iloc[:,-1]\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=102030)\n",
+    "\n",
+    "dtrain = xgb.DMatrix(X_train, label=y_train)\n",
+    "dtest = xgb.DMatrix(X_test, label=y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare params"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params = {'max_depth': 4,\n",
+    "          'eta': 0.7,\n",
+    "          'gamma': 0.1,\n",
+    "          'silent': 1,\n",
+    "          'subsample': 1,\n",
+    "          'lambda': 1,\n",
+    "          'alpha': 0.25,\n",
+    "          'objective': 'reg:linear',\n",
+    "          'eval_metric': ['mae', 'rmse']}\n",
+    "watchlist = [(dtest, 'eval'), (dtrain, 'train')]\n",
+    "num_round = 20"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train model using `xgb.train()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "neptune.create_experiment(name='xgb', tags=['train'], params=params)\n",
+    "xgb.train(params, dtrain, num_round, watchlist,\n",
+    "          callbacks=[neptune_callback(log_tree=[0,1,2])])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train model using `xgb.cv()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "neptune.create_experiment(name='xgb', tags=['cv'], params=params)\n",
+    "xgb.cv(params, dtrain, num_boost_round=num_round, nfold=7,\n",
+    "       callbacks=[neptune_callback(log_tree=[0, 1, 2, 3, 4])])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Train model using `sklearn` API"
```

Comment:
I'd add a screenshot or GIF of what people would see in the UI. 
Perhaps with an example link (just in case something didn't work)

Please add it for every new experiment you create.

Reply:
added gif for the overview and links to each experiment in the demo notebook.